enterprise/models.py

```diff
@@ -2554,6 +2555,28 @@ class SystemWideEnterpriseUserRoleAssignment(EnterpriseRoleAssignmentContextMixi
 
     role_class = SystemWideEnterpriseRole
 
+    enterprise_customer = models.ForeignKey(
+        EnterpriseCustomer,
+        null=True,
+        blank=True,
+        related_name="system_wide_role_assignments",
+        on_delete=models.CASCADE,
+        help_text=_(
+            'The EnterpriseCustomer for which the role is assigned for the user.  If creating a new assignment, '
+            'you must first provide a user email and role, then click "Save and continue editing" '
+            'BEFORE selecting from this dropdown.'
+        ),
+    )
+
+    applies_to_all = models.BooleanField(
```

Comment:
not something you need to change now, but curious about your thoughts on this field being something that is part of the parent model definition in edx-rbac? especially since the concept of the '*' for context already exists at that level, and this seems to be a more explicit way of defining that on the model?

Reply:
Hmm, yeah, that's a good idea.  I think it's not a ton of work to pull that out, lemme give it a shot.